### PR TITLE
#13: Implement Darknet19 backbone with pre-trained weight

### PR DESCRIPTION
**Pull request by visionNoob**
Darknet19 backbone 구현 

1. https://pjreddie.com/darknet/imagenet/ 의 table에 나와있는 것 처럼
256 x 256 x 3 input을 받습니다. 

2. 모델은 json으로 (caffe 스럽게...) 정의하도록 했습니다. (이 부분은 추후 논의가 필요합니다. )
(현재 json 구조는 제가 임의로 만들어서 확장성도 많이 떨어집니다. )
https://github.com/insurgent92/YOLO9000/blob/issue_13/darknet19.json

3. pretrained_weight는 https://github.com/marvis/pytorch-caffe-darknet-convert 를 사용해서
컨버팅했습니다. **일부 normalization 문제가 있습니다.** -> #33 

4. 컨버팅한 웨이트파일은 아마존 버킷에 올려놨습니다 -> #32

### CODE REVIEW

**Review by ssaru — APPROVED** [review]
완료했습니다.
